src/components/transfer/Transfer.tsx

```diff
@@ -112,6 +113,7 @@ const Transfer: React.FC<{
                 <div
                   className="link"
                   onClick={(): void => handleAddressClick(transfer.to)}
+                  style={{ color: '#d355e7' }}
```

Comment:
please, use class to define this as a pink color.
Can you change the className link to pink color? Or create a link-pink to define it.